cmake/SetupPackages.cmake

```diff
@@ -82,7 +82,7 @@ if (RAJA_ENABLE_HIP)
   endif()
 
   if (RAJA_ENABLE_EXTERNAL_ROCPRIM)
-    find_package(RocPRIM)
+    include(cmake/thirdparty/FindRocPRIM.cmake)
```

Comment:
@davidbeckingsale are you OK with this change? It eliminates a CMake warning about rocprim package name mismatch with newer CMake versions.